.prettierrc

```diff
@@ -2,5 +2,14 @@
   "bracketSpacing": false,
   "singleQuote": true,
   "printWidth": 80,
-  "trailingComma": "all"
+  "trailingComma": "all",
+  "overrides": [
+    {
+      "files": "**/*.md",
+      "options": {
+        "parser": "markdown",
+        "proseWrap": "always"
```

Comment:
Should it be `preserve`? See https://prettier.io/docs/en/options.html#prose-wrap

Reply:
`preserve` didn't enforce lines to be max 80 characters in width (which is what I'm trying to achieve with this PR). Hence went with `always`.